src/pages/api/login.ts

```diff
@@ -0,0 +1,53 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { v4 as uuidv4 } from 'uuid';
+import Auth from '../../data/amplify';
+import { getDomain, redirectTo, redirectToError, setCookieOnResponseObject, checkEmailValid } from './apiUtils';
+import { OPERATOR_COOKIE } from '../../constants';
+import { ErrorInfo } from '../../types';
+import { getOperatorNameByNocCode } from '../../data/auroradb';
+
+export default async (req: NextApiRequest, res: NextApiResponse): Promise<void> => {
+    try {
+        const { email, password } = req.body;
+
+        const errors: ErrorInfo[] = [];
+
+        if (!checkEmailValid(email)) {
+            errors.push({
+                id: 'email',
+                errorMessage: 'Enter an email address in the correct format, like name@example.com',
+            });
+            const cookieContent = JSON.stringify({ errors });
+            setCookieOnResponseObject(getDomain(req), OPERATOR_COOKIE, cookieContent, req, res);
+            redirectTo(res, '/login');
+        }
+
+        try {
+            const user = await Auth.signIn(email, password);
+
+            if (user) {
+                const nocCode = user.attributes['custom:noc'];
```

Comment:
We need to store the ID and Refresh tokens from this user response as a cookie, they're what we'll be using to check the user is authenticated

Reply:
ID and Refresh tokens now get set on USER_COOKIE.